x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/StatementParserTests.java

```diff
@@ -874,6 +875,44 @@ private void assertIdentifierAsIndexPattern(String identifier, String statement)
         assertThat(table.table().index(), is(identifier));
     }
 
+    public void testIdPatternUnquoted() throws Exception {
+        var string = "regularString";
+        assertThat(breakIntoFragments(string), contains(string));
+    }
+
+    public void testIdPatternQuoted() throws Exception {
+        var string = "`escaped string`";
+        assertThat(breakIntoFragments(string), contains(string));
+    }
+
+    public void testIdPatternQuotedWithDoubleBackticks() throws Exception {
+        var string = "`escaped``string`";
+        assertThat(breakIntoFragments(string), contains(string));
```

Comment:
👍 👍 👍 

nit: it would be good to test that the final field name will be ```escaped`string```, ie. that the double backtick is an escape for backtick inside quoted strings.

I couldn't find this detail in the docs, @leemthompo could you please double-check?

Reply:
Thanks for checking @leemthompo, that looks good, no need to add further information IMHO